students/dychowicz_monika/lesson_05/lesson_05.py

```diff
@@ -0,0 +1,99 @@
+def greater_than_neighbours():
```

Comment:
algorithm is broken.
does not output valid number for some inputs, e.g `5 4 3 2 1` (should be 0, not 3)